src/dataset.rs

```diff
@@ -164,6 +164,10 @@ impl Dataset {
         Ok(())
     }
 
+    pub fn spatial_ref(&self) -> Result<SpatialRef> {
+        unsafe { SpatialRef::from_c_obj(gdal_sys::GDALGetSpatialRef(self.c_dataset)) }
+    }
```

Comment:
Would it make sense to add something for GDALSetSpatialRef as well?

Reply:
I suppose so. And I think the CI will fail for GDAL 2.